distribution/lib/Standard/Base/0.0.0-dev/src/Errors/Common.enso

```diff
@@ -347,12 +347,19 @@ type Private_Access
        - this_project_name: Optional qualified name of the current (caller) project.
        - target_project_name: Optional qualified name name of the project of callee.
        - target_method_name: The name of the target method call that is project-private and thus cannot be called.
-    Error (this_project_name : (Text | Nothing)) (target_project_name : (Text | Nothing)) (target_method_name : Text)
+    private Error
+        (this_project_name : (Text | Nothing))
+        (target_project_name : (Text | Nothing))
+        (target_method_name : Text)
+        (msg : (Text | Nothing) = Nothing)
 
     ## PRIVATE
        Convert the Private_Access error to a human-readable format.
     to_display_text : Text
     to_display_text self =
+        if self.msg.is_nothing then self.default_to_text else self.msg
```

Comment:
btw. for future reference, this could be simplified to
```suggestion
        self.msg.if_nothing self.default_to_text
```

sorry I did not notice it before as this PR was pushed when I was off and it was already reviewed by others so I did not go too deep.

Of course there is no need to change that, as the current code is also good. Just letting you know about a nice idiom that can be used on similar occasions.

Reply:
- I was considering to use `if_nothing`
- but then I felt into more familiar `if_then_else`
- fixed in https://github.com/enso-org/enso/pull/13047/commits/6e475662dec271dde93ba237ca11ae904e32cb22